src/cmds/watch.js

```diff
@@ -30,10 +30,9 @@ module.exports = function watch() {
   nodeWatch('./', { 
     recursive: true,
     filter: (f, skip) => {
-      if (/\/node_modules/.test(f)) return skip;
-      if (/\/build/.test(f)) return skip;
```

Comment:
Why did you remove the build folder ?

Reply:
I noticed that those filters were not working because they were prefixed by a `/` which was not working for the root folders (`node_modules/`) instead of (`/node_modules/`). Then after this fix, I also noticed that `build` folder should not be filtered since this is there that we output our bundle file.